oxen-rust/docs/examples/1_InitAndCommit.md

```
@@ -2,7 +2,7 @@
 
 Oxen mirrors the commands from `git` that you know and love. This tutorial can serve as a refresher or a clean start if you have never used version control before.
 
-We will be working with an example dataset that you can download [here](https://github.com/Oxen-AI/Oxen/raw/main/data/datasets/SmallCatDog.zip). It is a smaller dataset created from the kaggle dataset found [here](https://www.kaggle.com/datasets/shaunthesheep/microsoft-catsvsdogs-dataset). It consists of 20 images of cats and dogs, along with some annotation files. 
+We will be working with an example dataset that you can download [here](https://github.com/Oxen-AI/Oxen/raw/main/data/datasets/SmallCatDog.zip). It is a smaller dataset created from the kaggle dataset found [here](https://www.kaggle.com/datasets/shaunthesheep/microsoft-catsvsdogs-dataset). It consists of 20 images of cats and dogs, along with some annotation files.
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)